test/library.spec.js

```diff
@@ -131,21 +131,7 @@ describe('createForm', () => {
       );
     });
 
-    it('returns false if some fields are untouched', async () => {
-      const touched = await subscribeOnce(instance.touched);
-      const someUntouched = Object.values(touched).some((val) => val === false);
-      expect(someUntouched).toBe(true);
-
-      const isValid = await subscribeOnce(instance.isValid);
-      expect(isValid).toBe(false);
-    });
-
-    it('returns true if form is valid and all fields touched', () => {
-      instance.touched.set({
-        name: true,
-        email: true,
-        country: true,
-      });
+    it('returns true if form is valid', () => {
       subscribeOnce(instance.isValid).then((isValid) =>
```

Comment:
@subpx the [build](https://travis-ci.com/github/tjinauyeung/svelte-forms-lib/jobs/457173622) is failing from an unhandled rejection, so it could likely be this line since there's no `.catch` statement.

This test should ideally use `async / await`, or `done()` should be called inside the `then` to signal that the test is complete, too.